env/main/templates/galaxy/config/job_conf_vgp.yml.j2

```diff
@@ -73,9 +73,9 @@ execution:
       function: map_tool_to_destination
       rules_module: tpv.rules
       tpv_config_files:
+        - "https://gxy.io/tpv/db.yml"
         - "{{ tpv_config_dir }}/defaults_vgp.yaml"
         - "{{ tpv_config_dir }}/environments_vgp.yaml"
-        - "https://raw.githubusercontent.com/galaxyproject/tpv-shared-database/main/tools.yml"
```

Comment:
If you cut regular releases we can create PRs for updates and test them out. That way we're getting timely updates without the surprise factor.